deploy/src/erc_to_erc/home.js

```diff
@@ -15,7 +15,7 @@ const {
 } = require('../deploymentUtils')
 const { web3Home, deploymentPrivateKey, HOME_RPC_URL } = require('../web3')
 
-const EternalStorageProxy = require('../../../build/contracts/EternalStorageProxy.json')
+const EternalStorageProxy = require('../../../build/contracts/ClassicEternalStorageProxy.json')
```

Comment:
I think this is from your local testing environment and is not needed so far for the `native-to-erc` bridge

Reply:
Thank you! Removed